mypy/main.py

```diff
@@ -457,9 +457,15 @@ def add_invertible_flag(flag: str,
         targets = []
         for f in special_opts.files:
             if f.endswith(PY_EXTENSIONS):
-                targets.append(BuildSource(f, crawl_up(f)[1], None))
+                try:
+                    targets.append(BuildSource(f, crawl_up(f)[1], None))
+                except ValueError as e:
```

Comment:
Can you change this to catch the `ValueError` only from `crawl_up(f)`? It would be even better if the latter function returned a special result (maybe `None`) or (even more better :-) raised a dedicated exception, instead of essentially crashing with `ValueError`. Ditto for `expand_dir(f)` below (though there at least the scope of the `except` clause is sufficiently limited).